golemcli/src/commands/acl.rs

```diff
@@ -400,3 +425,24 @@ async fn list(
         .to_response())
     }
 }
+
+fn warn_if_exist(default_rule: AclRule, direction: AclRule, exist: Option<Vec<String>>) {
```

Comment:
```suggestion
fn warn_if_exist(default_rule: AclRule, direction: AclRule, exist: Vec<String>) {
```

Reply:
and probably you could use borrowing to avoid copying arguments: https://doc.rust-lang.org/rust-by-example/scope/borrow.html

Reply:
done